website/src/widgets/quickstart-install.js

```diff
@@ -159,6 +159,9 @@ const QuickstartInstall = ({ id, title }) => {
                         setters={setters}
                         showDropdown={showDropdown}
                     >
+                        <QS os="mac" hardware="gpu" platform="arm">
+                            # Note M1 GPU support is experimental, see <a href="https://github.com/explosion/thinc/issues/792">this issue</a>
```

Comment:
I'd like link text looks more informative than basically "here", but I'm not particularly attached to this particular suggestion:

```suggestion
                            # Note M1 GPU support is experimental, see <a href="https://github.com/explosion/thinc/issues/792">Thinc issue #792</a>
```

Reply:
Looks good to me!